backend/tests/integration/ipam/test_load_concurrent_prefixes.py

```diff
@@ -0,0 +1,33 @@
+import ipaddress
```

Comment:
I think we can move this file to the functional tests directory instead 
Most integration tests needs to be migrated there